src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
```

Comment:
Consider using a more detailed block comment to describe the purpose and usage of the styles in this file 📝